apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/fix_sporadic_bloom_filter_jdbc_errors.sql

```diff
@@ -0,0 +1,10 @@
+/* 
+ BFD-1505
+ There are periodic JDBC errors related to the Loaded Filter Manager bloom filter implementation.
+ 
+ This corrects that issue by eliminating the sequence scan on loaded_batches with an index.
+ */
+ 
+CREATE INDEX IF NOT EXISTS loaded_batches_loaded_file_id_idx
```

Comment:
this should be:
CREATE INDEX CONCURRENTLY IF NOT EXISTS loaded_batches_loaded_file_id_idx

Reply:
Fixing